packages/components/addon/components/hds/tooltip/index.hbs

```diff
@@ -0,0 +1,5 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<button type="button" class="hds-tooltip" {{hds-tooltip this.text options=this.options}}>{{~yield~}}</button>
```

Comment:
I'm using a button for the opener element to start. This could be changed to `Hds::Interactive` if we like. 

One issue is that consumers could easily add invalid content inside it as it is now so we could either limit the type of content it accepts or else come up with a different structure so that their content won't be inside the button itself.

Reply:
Would like feedback on this.